### PR TITLE
Feat/48 create contactinfo component

### PR DESCRIPTION
**Pull request by Lingertje**

### CODE REVIEW

File: components/ProfileCard.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	const props = defineProps({
+		firstName: {
+			type: String,
+			required: true,
+		},
+		lastName: {
+			type: String,
+			required: true,
+		},
+		functionTitle: {
```
Comment (remypar5) [inline components/ProfileCard.vue:12]: Free info: the default value for `required` is `false` so you don't have to set is explicitly.

File: components/ProfileCard.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		},
+		email: {
+			type: String,
+			required: false,
+		},
+		telephone: {
+			type: String,
+			required: false,
+		},
+		imgUrl: {
```
Comment (remypar5) [inline components/ProfileCard.vue:24]: For consistency I'd go for `image`

```suggestion
		image: {
```

File: components/ProfileCard.vue [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+			<p>{{ props.functionTitle }}</p>
+			<p>{{ props.email }}</p>
+			<p>{{ props.telephone }}</p>
+		</div>
+	</div>
+</template>
+
+<style scoped>
+	.card {
+		@apply w-1/5 inline-block m-[1rem];
```
Comment (remypar5) [inline components/ProfileCard.vue:45]:
```suggestion
		@apply w-1/5 inline-block m-4;
```

File: components/ProfileCard.vue [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+		p {
+			@apply m-0 text-[0.8rem] leading-[1.4rem];
+		}
+
+		&:first-of-type {
```
Comment (remypar5) [inline components/ProfileCard.vue:51]: Does this actually work? They're scoped so I would expect them to not know about their surroundings

Comment (Lingertje) [reply components/ProfileCard.vue:51]: Good question, it actually does work. I looked into it and all the cards have the same data-attr `.card[data-v-57c66cfd]`. It's using that data-attr to add the `:first-of-type`

File: components/ProfileCard.vue (hunk 3 above)
Comment (Lingertje) [reply components/ProfileCard.vue:45]: Good to know!

File: components/ProfileCard.vue [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+			type: String,
+			required: false,
+		},
+	})
+</script>
+
+<template>
+	<div class="card">
+		<img v-if="props.imgUrl" :src="props.imgUrl">
+		<div class="card-body">
```
[the remark this answers is not in the document]
Comment (remypar5) [reply components/ProfileCard.vue:25]: Sorry, I meant the whole `.card-body` should be an `address` element

**Review by remypar5 — CHANGES_REQUESTED** [review]

File: components/ProfileCard.vue (hunk 5 above)
[the remark this answers is not in the document]
Comment (Lingertje) [reply components/ProfileCard.vue]: Check, i see

**Review by remypar5 — APPROVED** [review]

**Pull request merged**